src/raw/client.rs

```diff
@@ -163,9 +162,6 @@ impl Client {
         if let Some(count) = count {
             request = request.param("count", count);
         }
-        if let Some(time_range) = time_range {
-            request = request.param("time_range", time_range);
-        }
 
```

Comment:
Justification for this removal? Please justify in the commit.

Reply:
It's been removed from the actual API endpoint, but yeah clarifying in the commit message would be nice

Reply:
True, I should have justified it, and will try to improve my messages from now on.

As for why, it's actually unused. Both documentation and testing shows that there's no "range" nor "time_range" parameter for the request. I also edited the test to remove the parameter later on

If you want to test it by yourself:
https://api.listenbrainz.org/1/user/rustynova/listens?count=999&time_range=this_week

Just change my username with yours, but I'm not doing 999 listens per week, while 999 listens are returned

Reply:
Alright, no worries. Let's resolve this convo.